老國音.js

```diff
@@ -0,0 +1,291 @@
+/* 推導老國音
+
+音系：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E9%9F%B3%E7%B4%A0
+音變規則：https://zh.wikiversity.org/zh-hant/%E8%80%81%E5%9C%8B%E9%9F%B3%E8%88%87%E5%BB%A3%E9%9F%BB%E5%B0%8D%E6%AF%94
+字音來源：https://github.com/baopaau/rime-bepemefeve/blob/main/bepemefeve.dict.yaml
+注意「僞老國音」：https://www.bilibili.com/read/cv17377530/
+
+@author 307587 */
+
+/** @type { 音韻地位['屬於'] } */
+const is = (...x) => 音韻地位.屬於(...x);
+/** @type { 音韻地位['判斷'] } */
+const when = (...x) => 音韻地位.判斷(...x);
+
+if (!音韻地位) return [
+  ['注音符號\n否的話就用國際音標', true],
+  ['ㄬㄭ|ㄬ、ㄭ音標\n啟用「注音符號」時無效', [2, 'ȵ、ɿ、ʅ', 'ɲ、ɹ̩、ɻ̍']],
+  ['介母', [1,
+    { text: '元音：i u y', value: '元音' },
+    { text: '半元音：j w ɥ', value: '半元音' },
+  ], { hidden: 選項.注音符號 === true }],
+  ['ㄚ|ㄚ元音\nㄧㄞ：寬式/iai/、嚴式[iæi]\nㄧ、ㄩ後的ㄢ：寬式/-an/、嚴式[-ɛn]', [2, '嚴式', '寬式']],
```

Comment:
```suggestion
  ['ㄚ|ㄧㄞ ㄧㄢ ㄩㄢ', [1, 'iai ian yan', 'iæi iɛn yɛn'], { hidden: 選項.注音符號 !== false }],
```

这个选项叫ㄚ元音不合适，这里面没有“ㄚ”。不妨直接这样写，还避免了对严式宽式的判定